kuma/wiki/kumascript.py

```diff
@@ -40,20 +38,18 @@ def should_use_rendered(doc, params, html=None):
             (force_macros or (not no_macros and not show_raw)))
 
 
-def post(request, content, locale=settings.LANGUAGE_CODE):
+def _post(content, env_vars, timeout=config.KUMASCRIPT_TIMEOUT):
```

Comment:
This file contains the main changes I made to Kuma as part of this KumaScript refactor: KumaScript no longer supports a GET endpoint: everything has to be done with POST. So this file is changed so that the get() method actualliy does a POST request. (I punted on changing the method name, for now.)